content/quiz_events.md

```diff
@@ -95,6 +95,40 @@ Below are the the scoring calculation rules and the logic for them. This procedu
 **Team Roster**
 : This is an array of quizzer objects, each of which contains a "correct answers" value
 
+### How to Score Individual and Team Points
+**Individual Points**
+
+_**Points Earned**_
+- +20 Points for every correct question & toss-up question
+- +10 Points for every quiz out without error
+
+**_Points Deducted_**
+- -10 Points for 2nd and subsequent personal errors
+     - Points are not deducted for errors on bonus questions
+     - Points are not deducted for errors during overtime
```

Comment:
If you're stating this for errors, you should state that the same applies for correct questions during overtime. 

OR, higher up state that "overtime events have no individual scoring effect"

Reply:
I like the higher up statement. I would write. 

All points that occur during a bonus question or during overtime do not contribute to the individual score of a quizzer.